docs(config): add vscode debug error tip (#19631)
vitejs/vite: docs/config/index.md

  }
})
```

> ## VScode Config

Reviewer: ```suggestion
## Debugging a config file on VScode
```

> ## Debugging a config file on VScode

Reviewer 2: ```suggestion
## Debugging the Config File on VS Code
```
We use title case for the headings, so some words start with a capital letter. VS Code also has a weird spelling with a space in the middle.

Author: Looks great.

> Vite 6 writes the generated temporary configuration file to the `node_modules/.vite-temp` folder, a file not found error will occur when setting breakpoint debugging in the vite config file. Adding the following configuration to the vscode configuration file can solve the problem.

Reviewer: ```suggestion
With a default `--configLoader bundle`, Vite 6 writes the generated temporary configuration file to the `node_modules/.vite-temp` folder, a file not found error will occur when setting breakpoint debugging in the vite config file. Adding the following configuration to the vscode configuration file can solve the problem.
```

Author: Thanks for the suggestion, it looks great. 👍

> With a default `--configLoader bundle`, Vite 6 writes the generated temporary configuration file to the `node_modules/.vite-temp` folder and a file not found error will occur when setting breakpoint debugging in the vite config file. Adding the following configuration to the vscode configuration file can solve the problem.

Reviewer 2: ```suggestion
With the default `--configLoader bundle` behavior, Vite writes the generated temporary configuration file to the `node_modules/.vite-temp` folder and a file not found error will occur when setting breakpoint debugging in the Vite config file. To fix the issue, add the following configuration to `.vscode/settings.json`:
```

I reworded here a bit to remove "Vite 6" since the docs should already be about vite 6. I also updated the last sentence to show the settings file instead so they know where to place it.

Author: Thanks, this looks clearer.
